docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+Reference to the permission mapper to use to the right of the operation.
+
+swarm.elytron.logical-role-mappers._KEY_.left:: 
+Reference to a role mapper to be used on the left side of the operation.
+
+swarm.elytron.logical-role-mappers._KEY_.logical-operation:: 
+The logical operation to be performed on the role mapper mappings.
+
+swarm.elytron.logical-role-mappers._KEY_.right:: 
+Reference to a role mapper to be used on the right side of the operation.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.delegate-realm-mapper:: 
+The RealmMapper to delegate to if the pattern does not match. If no delegate is specified then the default realm on the domain will be used instead. If the username does not match the pattern and a delegate realm-mapper is present, the result of delegate-realm-mapper is mapped via the realm-map.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.pattern:: 
+The regular expression which must contain at least one capture group to extract the realm from the name. If the regular expression matches more than one capture group, the first capture group is used.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.realm-map:: 
+Mapping of realm name extracted using the regular expression to a defined realm name. If the value for the mapping is not in the map or the realm whose name is the result of the mapping does not exist in the given security domain, the default realm is used.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.available-mechanisms:: 
+The SASL mechanisms available from this factory after all filtering has been applied.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.enabling:: 
+When set to 'true' no provider loaded mechanisms are enabled unless matched by one of the filters, setting to 'false' has the inverse effect.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.filters:: 
+The filters to apply when comparing the mechanisms from the providers, a filter matches when all of the specified values match the mechanism / provider pair.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.sasl-server-factory:: 
+Reference to a sasl server factory to be wrapped by this definition.
+
+swarm.elytron.periodic-rotating-file-audit-logs._KEY_.attribute-synchronized:: 
+Whether every event should be immediately synchronised to disk.
+
+swarm.elytron.periodic-rotating-file-audit-logs._KEY_.format:: 
+The format to use to record the audit event.
+
+swarm.elytron.periodic-rotating-file-audit-logs._KEY_.path:: 
+Path of the file to be written.
+
+swarm.elytron.periodic-rotating-file-audit-logs._KEY_.relative-to:: 
+The relative path to the audit log.
+
+swarm.elytron.periodic-rotating-file-audit-logs._KEY_.suffix:: 
+The suffix string in a format which can be understood by java.time.format.DateTimeFormatter. The period of the rotation is automatically calculated based on the suffix.
```

Comment:
Java class path should have some formatting. I lean towards backticks

Reply:
Note to self: not easily fixable in our sources.